src/RadixDlt.NetworkGateway.PostgresIntegration/DbQueryExtensions.cs

```diff
@@ -192,7 +193,8 @@ public static IQueryable<LedgerTransaction> GetFirstLedgerTransactionAtEpochRoun
         return dbContext
             .LedgerTransactions
             .Where(lt => lt.Epoch >= epoch && lt.RoundInEpoch >= round && lt.IndexInRound == 0)
-            .OrderBy(lt => lt.StateVersion)
+            .OrderBy(lt => lt.Epoch)
+            .ThenBy(lt => lt.RoundInEpoch)
```

Comment:
Might be worth a comment that this should be fully specified due to the `lt.IndexInRound = 0`? (And associated partial index).

Reply:
No longer relevant as we've dropped ORDER BY clause altogether - we rely on unique index anyways.